src/com/boggle/Main.java

```diff
@@ -6,8 +6,267 @@
 
     public static void main(String[] args) {
 
-        System.out.println(checkString("dog"));
-        System.out.println(checkString("return"));
+       explore(0,0, "");
+
+
+    }
+
+    public static String position(int row, int col){
+        String corner = "corner";
+        String middle = "middle";
+        String side = "side";
+        String actual = "";
+
+        if (row == 0 && col == 0 || row == 0 && col == 4 || row == 4 && col == 0 || row == 4 && col == 4){
+            actual = corner;
+        }
+
+        else if (row == 1 && col == 1 || row == 1 && col == 2 || row == 1 && col == 3){
+            actual = middle;
+        }
+
+        else if (row == 2 && col == 1 || row == 2 && col == 2 || row == 2 && col == 3) {
+            actual = middle;
+        }
+
+        else if (row == 3 && col == 1 || row == 3 && col == 2 || row == 3 && col == 3){
+            actual = middle;
+        }
+
+        else {
+            actual = side;
+
+            if (row == 0 && col == 1 || row == 0 && col == 2 || row == 0 && col == 3){
+                actual += " Upper";
+            }
+            else if (row == 0 && col == 1 || row == 0 && col == 2 || row == 0 && col == 3){
+                actual += " Left";
+            }
+            else if (row == 4 && col == 1 || row == 4 && col == 2 || row == 4 && col == 3){
+                actual += " Right";
+            }
+            else {
+                actual += " Lower";
+            }
+        }
+
```

Comment:
these if statements are kind of gross though, wouldn't you agree? 
in boggle, to explore a tile all you need to do is find the adjacent tiles right, can you think of an easier way to do this?

Reply:
hint: isn't all you need to check if an index is at 0 or 4 (i.e. you cant explore up, down, left or right)

Reply:
Once the program reaches the point at if if thinks the position is a certain side, it knows that it cannot be a corner or a middle. So, to make the code simpler, if the row is 0, it will automatically be an upper side, if the col is 0, it will be a left side, it the row is 4, it will be the lower side, and if the col is 4, it will be the right side.